manifests/config/v2.pp

```diff
@@ -20,12 +18,10 @@
       loglevel => warning,
     }
 
-    if $_securitycheck == undef {
-      $_securitycheck = false
-    }
+    unless defined('$_securitycheck') { $_securitycheck = true }
   }
 
-  if $_securitycheck == false and $::secc_snmpd::enforce_password_security == true {
+  if defined('$_securitycheck') and $_securitycheck and $::secc_snmpd::enforce_password_security {
```

Comment:
Are these two checks still necessary with puppet?
`defined('$_securitycheck') and $_securitycheck`

Reply:
yep, it is:
* removing `defined('$_securitycheck')` from the if clause brings back the warning:
  `  Warning: Unknown variable: '_securitycheck'. at /etc/puppetlabs/code/modules/secc_snmpd/manifests/config/v2.pp:24:6`